src/CAPathSetup.cpp

```diff
@@ -0,0 +1,79 @@
+// SPDX-License-Identifier: BSD-2-Clause
+//
+// This code has been produced by the European Spallation Source
+// and its partner institutes under the BSD 2 Clause License.
+//
+// See LICENSE.md at the top level for license information.
+//
+// Screaming Udder!                              https://esss.se
+
+#include "CAPathSetup.h"
+#include <cstdlib>
+#include <fstream>
+
+#ifdef _WIN32
+#include <direct.h>
+#include <stdlib.h>
+#else
+#include <unistd.h>
+#endif
+
+bool fileExists(std::string const &FullPath) {
+  std::fstream InFile(FullPath);
+  return InFile.good();
```

Comment:
I'm still looking into the failure in the system tests, but hit this issue first.
For some reason I didn't have write permissions over caRepeater so fileExists always fails.
Using `fopen` would avoid this problem:
```suggestion
  if (FILE *File = fopen(FullPath.c_str(), "r")) {
    fclose(File);
    return true;
  } else {
    return false;
  }
```
and change `#include <fstream>` to `#include <stdexcept>`.

Reply:
In kafkacow we resorted to boost: https://github.com/ess-dmsc/kafkacow/blob/master/src/GetSchemaPath.cpp but if we can make your approach work consistently then it is probably the lesser of the evils.
When the mac nodes are finally updated we can use `std::filesystem`: https://github.com/ess-dmsc/kafkacow/pull/109

Reply:
Yes, I am looking forward to being able to use `std::filesystem`. In this particular case, I am confident that using `std::ifstream` is a better option than using `fopen(..., "r")` however. Please test with the updated code.